tests/python/pants_test/pants_run_integration_test.py

```diff
@@ -327,6 +328,8 @@ def run_pants_with_workdir_without_waiting(self, command, workdir, config=None,
         ini.write(fp)
       args.append('--pants-config-files=' + ini_file_name)
 
+    Path('pyproject.toml').touch()
```

Comment:
Instead of this, add `//:pyproject` to the requirements for `pants.pex` here:

https://github.com/pantsbuild/pants/blob/f673fd2a4f59ea310f24ed702a01779118dad88f/src/python/pants/bin/BUILD#L71-L90

Reply:
I had tried this better and it didn't work because the buildroot is not guarenteed to be the same. Fixing to take your suggestion into account and do "the right thing"